howto_resize_image.md

```diff
@@ -0,0 +1,14 @@
+# How to resize image
+
+To make an image resize, we are using the method `load` of the class [Image](https://reference.aspose.com/imaging/net/aspose.imaging/image/) to load an image. Then we [resize](https://reference.aspose.com/imaging/net/aspose.imaging/image/resize/) the image to a new size by specifying desired height and width `300x300` in pixels and save the result to a new file. 
+
+Example C# code:
```

Comment:
@anms23 , I see that child pages does not contain description and keywords, is it expected?

Reply:
I've added keywords and descriptions to the child pages:
https://github.com/aspose-imaging/aspose-imaging-probe/commit/be1d29f5c6fbd91e3d24c185edbd640245caf7e9